external-crates/move/move-compiler/transactional-tests/tests/dependencies/public_package_different_packages.move

```diff
@@ -0,0 +1,17 @@
+//#init --edition 2024.alpha
+
+//# publish
```

Comment:
These will not actually be in different packages IIRC. What do you want this test to check?

Reply:
It's modelled after this one: https://github.com/MystenLabs/sui/blob/main/external-crates/move/move-bytecode-verifier/transactional-tests/tests/dependencies/access_private_function.mvir

Reply:
Yeah, but this one _should_ work as far as the abstractions the compiler is setting forth are. 

It is a bit wonky, so I might change this after this goes in